test/k8s-integration/main.go

```diff
@@ -100,14 +101,16 @@ func handle() error {
 			}
 		}()
 
-		*stagingImage = fmt.Sprintf("gcr.io/%s/gcp-persistent-disk-csi-driver", project)
+		if !*skipDriverBuild {
```

Comment:
lot of double negatives here. Looks like every time we use `skipDriverBuild` we do `!skipDriverBuild`, lets turn all these double negatives into a positive and just use `doDriverBuild` or something.

Reply:
done